README.md

```diff
@@ -1,21 +1,83 @@
-- Mailing list: [Google Groups](http://groups.google.com/group/terraform-tool)
+[![alt text](https://cdn.rawgit.com/hashicorp/terraform-website/master/content/source/assets/images/logo-hashicorp.svg)](https://www.terraform.io/)
 
-<img src="https://cdn.rawgit.com/hashicorp/terraform-website/master/content/source/assets/images/logo-hashicorp.svg" width="600px">
+Quick Start
+-----------
+> When using the OpenTelekomCloud Provider with Terraform 0.13 and later, the recommended approach is to declare Provider versions in the root module Terraform configuration, using a `required_providers` block as per the following example. For previous versions, please continue to pin the version within the provider block.
+
+1. Add [opentelekomcloud/opentelekomcloud](https://registry.terraform.io/providers/opentelekomcloud/opentelekomcloud/latest/docs) to your `required_providers`.
+    ```hcl
+    # provider.tf
+    terraform {
+      required_providers {
+        opentelekomcloud = {
+          source = "opentelekomcloud/opentelekomcloud"
+          version = ">= 1.23.2"
+        }
+      }
+    }
+    ```
+
+2. Run `terraform init -upgrade` to download the provider.
+3. Add the provider and supply your `tenant_name` and `domain_name` for minimum configuration.
+    ```hcl
+    # provider.tf
+    provider "opentelekomcloud" {
+      # OpenTelekomCloud Provider Documentation:
+      # https://registry.terraform.io/providers/opentelekomcloud/opentelekomcloud/latest/docs
+      # domain_name = "..."
+      # tenant_name = "..."
+      # auth_url    = "https://iam.eu-de.otc.t-systems.com/v3"
+      # user_name   = "..."
+      # password    = "..."
+    }
+    ```
+5. [Authenticate](https://registry.terraform.io/providers/opentelekomcloud/opentelekomcloud/latest/docs#authentication) either by providing `user_name` and `password` in the previous file or setting them as environment variables. 
+     ```bash
+     # Linux
+     OS_USERNAME="<your_username>"
+     OS_PASSWORD="<your_password"
+     # Windows
+     $env:OS_USERNAME="<your_username>"
+     $env:OS_PASSWORD="<your_password"
+     ```
+7. Create your first resource. 
+     ```hcl
+     # main.tf
+
+     # Create an Elastic Cloud Server resource
+     resource "opentelekomcloud_compute_instance_v2" "test-server" {
+       name        = "test-server"
+       image_name  = "Standard_CentOS_8_latest"
+       flavor_name = "s3.medium.1"
+       key_pair        = "kp_ecs"
+       security_groups = ["default"]
+       network {
+         name = "network_ecs"
+       }
+     }
+     ```
+
+### Full Examples
+
+ - [Have a look here for basic examples](https://github.com/opentelekomcloud/terraform-provider-opentelekomcloud/tree/devel/examples/basic-examples/modules)
+ - [and here for more advanced examples](https://github.com/opentelekomcloud/terraform-provider-opentelekomcloud/tree/master/examples)
+
+Don't forget to fill in the required variables in `variables.tf`.
```

Comment:
In most cases for `basic-examples` name of the file is `var.tf`. Maybe don't specify the file name here?